web-share/resources/manual-helper.js

```diff
@@ -0,0 +1,71 @@
+// Sets up the page for running manual tests. Automatically creates the
+// instructions (based on the parameters) and the share button.
+function setupManualShareTest(
```

Comment:
Ok, but the alternative is to be a bit more clear with a comment when "setupManualShareTest" is called.